aisynphys/layer_depths.py

```diff
@@ -0,0 +1,155 @@
+import numpy as np
+from neuron_morphology.lims_apical_queries import get_data
+from neuron_morphology.transforms.pia_wm_streamlines.calculate_pia_wm_streamlines import get_depth_gradient_field
+from neuron_morphology.snap_polygons.__main__ import run_snap_polygons, Parser
+from neuron_morphology.snap_polygons.types import ensure_path, ensure_linestring, ensure_polygon
+import neuron_morphology.layered_point_depths.__main__ as ld 
+from shapely.geometry import Polygon, Point, LineString, LinearRing
+import logging
```

Comment:
can these be made optional?

Reply:
they could, but i just made the import of layer_depths from the pipeline module optional instead, seemed simpler and accomplishes the same thing